templates/github/.github/workflows/nightly.yml.j2

```diff
@@ -182,16 +189,16 @@ jobs:
         with:
           python-version: "3.7"
 
-      - uses: actions/setup-ruby@v1
-        with:
-          ruby-version: "2.6"
-
       - name: Install httpie
         run: |
           echo ::group::HTTPIE
           sudo apt-get update -yq
           sudo -E apt-get -yq --no-install-suggests --no-install-recommends install httpie
           echo ::endgroup::
+          echo "TEST=performance" >> $GITHUB_ENV
```

Comment:
@dkliban when it is not iterable, this line is everything it needs, here you can see the test: https://github.com/fao89/pulp_file/runs/1557393762?check_suite_focus=true